website_team/README.rst

```diff
@@ -24,20 +24,12 @@ Maintainers
 Further information
 ===================
 
-Demo: http://runbot.it-projects.info/demo/??
+Demo: http://runbot.it-projects.info/demo/addons-dev/website-addons-8.0-website_team_new
```

Comment:
You shall not use addons-dev
https://odoo-development.readthedocs.io/en/latest/dev/docs/README.rst.html#addons-dev